01-JS-DOM-APIs/Exercises1_2/js/main.js

```diff
@@ -0,0 +1,12 @@
+window.onload = function(a)
+{
+    hidden();
+    document.getElementById("btn").addEventListener("click", function showAlert(){
+        alert("You push!");
+    });
+    
+};
+
+function hidden(){
```

Comment:
**fade-in**: a film-making and broadcasting technique whereby an image is made to appear gradually or the volume of sound is gradually increased from zero.

This is not working as expected. You should try to accomplish using css.

Reply:
https://i.stack.imgur.com/aKiOo.gif